recipes/mender-client/debian-master/control

```diff
@@ -1,24 +1,41 @@
 Source: mender-client
-Maintainer: Mender Team <mender@northern.tech>
+Maintainer: The Mender Team <mender@northern.tech>
 Section: admin
 Priority: optional
-Standards-Version: 3.9.2
-Build-Depends: debhelper (>= 9)
+Standards-Version: 4.0.0
+Depends: mender-update, mender-auth
+Build-Depends-Indep: cmake, debhelper (>= 10)
+Build-Depends: liblmdb-dev (>= 0.9.7), libc6-dev (>= 2.14), libboost-dev (>= 1.67.0), libboost-filesystem-dev (>= 1.67.0), libboost-log-dev (>= 1.67.0), libboost-regex-dev (>= 1.67.0), libboost-chrono-dev (>= 1.67.0), libboost-thread-dev (>= 1.67.0), libboost-atomic-dev (>= 1.67.0),  libssl-dev (>= 1.1.1), libarchive-dev (>= 3.2.2)
```

Comment:
This was added mostly due to `mk-build-deps`, but we can decide what we want to do with this.

Reply:
It's ok, IMHO.